org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/ast/ExprQt.java

```diff
@@ -46,6 +46,8 @@
  * <br>
  * <b>Invariant:</b> type!=EMPTY => sub.mult==0 <br>
  * <b>Invariant:</b> type!=EMPTY => vars.size()>0
+ *
+ * @modified Eduardo Pessoa, Nuno Macedo // [HASLab] electrum-temporal
```

Comment:
just adapts the creation of the `Decl`, always immutable (null `isVar`) in this context